### PR TITLE
Request an EC2 to run cvat shiny app

### PR DESCRIPTION
**Pull request by xindiguo**

### CODE REVIEW

File: config/prod/xind-cvat-test.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  AMIId: "ami-0ca7208f904dc1874"      # AWS linux 2 AMI 2.0.20200108 x86_64 HVM gp2
+  # AMIId: "ami-0acb259f7a7b035f6"      # Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-02-12
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  InstanceType: "t3.xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "50"
```
Comment (zaro0508) [inline config/prod/xind-cvat-test.yaml]: are you sure this disk size is big enough?

File: config/prod/xind-cvat-test.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "xindi.guo@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "ami-0ca7208f904dc1874"      # AWS linux 2 AMI 2.0.20200108 x86_64 HVM gp2
+  # AMIId: "ami-0acb259f7a7b035f6"      # Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-02-12
```
Comment (zaro0508) [inline config/prod/xind-cvat-test.yaml]: you need to comment out the aws linux and enable this one.  the ID is incorrect in the example, it should be `ami-0d576d466a205f6db`.  please update.

File: config/prod/xind-cvat-test.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: cvat-ubuntu
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["80","8082"]
```
Comment (zaro0508) [inline config/prod/xind-cvat-test.yaml]: do you need this instance to be accessible by people outside of sage?

**Comment by thomasyu888** [comment]
See resolved conversations on this one: https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/158/files

File: config/prod/xind-cvat-test.yaml (hunk 1 above)
Comment (xindiguo) [reply config/prod/xind-cvat-test.yaml]: Yes, this is just for testing. We won't put too many video files on there

File: config/prod/xind-cvat-test.yaml (hunk 3 above)
Comment (xindiguo) [reply config/prod/xind-cvat-test.yaml]: No, not this time

File: config/prod/xind-cvat-test.yaml (hunk 1 above)
Comment (xindiguo) [reply config/prod/xind-cvat-test.yaml]: I updated it to 200 just in case.

**Pull request merged**